app/serializers/spree/v2/storefront/wished_product_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  module V2
+    module Storefront
+      class WishedProductSerializer < BaseSerializer
+        set_type :wished_product
+
+        attributes :id, :remark
```

Comment:
@chhlga please do not include `id` in the attributes list, however, we definitely need quantity